templates/download/iot/index.html

```diff
@@ -50,62 +50,60 @@ <h2 class="p-heading-icon__title" style="line-height: 4rem;">适用于树莓派
         }}
       </div>
     </section>
-      
-    <section class="p-strip">
+
+    <section class="p-strip is-deep">
       <div class="row">
-        <div class="col-8">
+        <div class="col-2 col-medium-4">
```

Comment:
```suggestion
        <div class="col-2 col-medium-4 u-sv3">
```
To add a little spacing between this heading and the content on medium-small screens

Reply:
Thanks, added.